components/clm/src/main/clm_varctl.F90

```diff
@@ -213,7 +213,9 @@ module clm_varctl
   !----------------------------------------------------------
 
   logical, public            :: use_fates = .false.              ! true => use  ED
-  logical, public            :: use_fates_spitfire = .false.  ! true => use spitfire model
+  integer, public            :: fates_spitfire_mode = 0                ! 0 for no fire; 1 for constant ignitions
+  logical, public            :: use_fates_fixed_biogeog = .false.           ! true => use fixed biogeography mode
+  logical, public            :: use_fates_spitfire = .false.           ! true => use spitfire model
```

Comment:
It seems that `use_fates_spitfire` has been replaced by `fates_spitfire_mode`. If true, can `use_fates_spitfire` be deleted?

Reply:
yes, good catch @bishtgautam

Reply:
updated code